iteration/iteration_test.go

```diff
@@ -458,84 +450,51 @@ func (s *TestIterationRepository) TestExistsIteration() {
 
 func (s *TestIterationRepository) TestIsActive() {
 	t := s.T()
+	newSpace := space.Space{
+		Name:    testsupport.CreateRandomValidTestName("Space Exists"),
+		OwnerId: s.testIdentity.ID,
+	}
+	repoSpace := space.NewRepository(s.DB)
+	space, err := repoSpace.Create(context.Background(), &newSpace)
+	assert.Nil(t, err)
 	t.Run("user active is true", func(t *testing.T) {
-		// given
-		newSpace := space.Space{
-			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
-			OwnerId: s.testIdentity.ID,
-		}
-		repoSpace := space.NewRepository(s.DB)
-		space, err := repoSpace.Create(context.Background(), &newSpace)
-		assert.Nil(t, err)
-
 		level0IterationName := "Top level iteration"
-		trueUserActive := true
 		i1 := iteration.Iteration{
 			Name:       level0IterationName,
 			SpaceID:    space.ID,
-			UserActive: &trueUserActive,
+			UserActive: true,
 		}
 		require.True(t, i1.IsActive())
 	})
 	t.Run("start date is nil", func(t *testing.T) {
-		// given
-		newSpace := space.Space{
-			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
-			OwnerId: s.testIdentity.ID,
-		}
-		repoSpace := space.NewRepository(s.DB)
-		space, err := repoSpace.Create(context.Background(), &newSpace)
-		assert.Nil(t, err)
-
 		level0IterationName := "Top level iteration"
-		falseUserActive := false
 		i1 := iteration.Iteration{
 			Name:       level0IterationName,
 			SpaceID:    space.ID,
-			UserActive: &falseUserActive,
+			UserActive: false,
 		}
 		require.False(t, i1.IsActive())
 	})
 	t.Run("end date is nil and current date is after start date", func(t *testing.T) {
-		// given
-		newSpace := space.Space{
-			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
-			OwnerId: s.testIdentity.ID,
-		}
-		repoSpace := space.NewRepository(s.DB)
-		space, err := repoSpace.Create(context.Background(), &newSpace)
-		assert.Nil(t, err)
-
 		start := time.Now()
 		level0IterationName := "Top level iteration"
-		falseUserActive := false
 		i1 := iteration.Iteration{
 			Name:       level0IterationName,
 			SpaceID:    space.ID,
 			StartAt:    &start,
-			UserActive: &falseUserActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c